PoseMsg task_frame
float64 speed_factor
float64[6] direction
float64[6] fd
float64 timeout
float64 f_thresh
franka_example_controllers/Gain controller_gain
